packages/openapi3-parser/test/unit/parser/oas/parseServerVariableObject-test.js

```diff
@@ -62,22 +63,22 @@ describe('#parseServerVariableObject', () => {
         description: 1234,
       });
 
-      const parseResult = parse(context)(serverVariable);
-      expect(parseResult.get(0)).to.be.instanceof(namespace.elements.String);
+      const parseResult = parse(context)(serverVariable, 'name');
+      expect(parseResult.get(1)).to.be.instanceof(namespace.elements.Annotation);
```

Comment:
Fwiw, `expect(parseResult).to.contain.warning` (line below) checks that there is an annotation, and its class contains `warning`. The old line was testing that the first element is the valid element being returned (the precense of the warning still results in the first item being the expected value as apposed to a different warning or error). Generally all of the warning tests are written in same way to protect against the first value returning anything but a valid value.

Reply:
Kept the check for the valid entry in there.